index.js

```diff
@@ -25,9 +25,14 @@ module.exports = function (opts, minimistOpts) {
 			cwd: parentDir,
 			normalize: false
 		}).pkg,
-		argv: process.argv.slice(2)
+		argv: process.argv.slice(2),
+		inferType: false
 	}, opts);
 
+	if (opts.inferType === false) {
+		minimistOpts = objectAssign(minimistOpts || {}, {string: ['_']});
```

Comment:
This will override any `string` config the user does. We should instead make it foolproof by ensuring `string` always contains `'_'`.